EIPS/eip-7791.md

```diff
@@ -0,0 +1,102 @@
+---
+eip: 7791
+title: GAS2ETH opcode
+description: Introduces a new opcode, `GAS2ETH`, to convert gas/mana to ETH
```

Comment:
```suggestion
description: Introduces a new opcode, `GAS2ETH`, to convert gas to ETH
```

Reply:
I think we should keep "gas/mana" to keep it future-proof:
- https://github.com/ethereum/EIPs/pull/6789
- https://github.com/vyperlang/vyper/pull/3713